tests/misc/tests/misc.js

```diff
@@ -6,6 +6,7 @@ const {
   TOKEN_PROGRAM_ID,
   Token,
 } = require("@solana/spl-token");
+const miscIdl = require("../target/idl/misc.json");
```

Comment:
I had to import the idl manually because I wanted to create an account with a different wallet, so I had to create a whole new program. If you know a better way, I'm all ears.